src/managers/KeybindManager.cpp

```diff
@@ -1966,8 +1967,16 @@ void CKeybindManager::mouse(std::string args) {
             g_pKeybindManager->m_bIsMouseBindActive = true;
 
             g_pInputManager->currentlyDraggedWindow = g_pCompositor->vectorToWindowIdeal(g_pInputManager->getMouseCoordsInternal());
-            g_pInputManager->dragMode               = MBIND_RESIZE;
 
+            if (isNumber(EXTRARG)) {
+                switch (stoi(EXTRARG)) {
```

Comment:
a) `std::stoi`
b) all `sto`s have to be wrapped in a try{} to prevent bad data crashes. I recommend doing this in the beginning and ignoring it if it's invalid.